PetAdoptions/cdk/pet_stack/lib/services/status-updater-service.ts

```diff
@@ -19,6 +25,20 @@ export class StatusUpdaterService extends Construct {
   constructor(scope: Construct, id: string, props: StatusUpdaterServiceProps) {
     super(scope, id);
 
+    // create the S3 Bucket for the Lambda testing
```

Comment:
This is FIS Bucket. Not the Status Update. Move this out please. We may have additional functions subject to experiment. 
would be good to preserve bucket details in SSM.